monkey/monkey_island/cc/services/reporting/report.py

```diff
@@ -274,7 +274,7 @@ def get_exploits():
         for exploit in mongo.db.telemetry.aggregate(query):
             new_exploit = ReportService.process_exploit(exploit)
             if new_exploit not in exploits:
-                exploits.append(new_exploit)
+                exploits.append(new_exploit.__dict__)
```

Comment:
Why? You have a wrapper function in dataclasses to cast to dict

Reply:
AFAIK that wrapper only calls factory function, which in our case is __dict__ anyways. I see no reason to use that wrapper